website_menu_icon/models/website_menu.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Menu(models.Model):
+
+    _inherit = "website.menu"
+
+    image = fields.Char("Icon", required=False, translate=True)
+    image_type = fields.Selection(
+        [("none", "None"), ("image", "Image"), ("icon", "Icon")],
```

Comment:
Since the field is not required, `False` is a valid value. Then what is `none` for? Better just remove it:

```suggestion
        [("image", "Image"), ("icon", "Icon")],
```